### ISSUE TITLE
Fails inside { ... } block

### ISSUE THREAD

**Issue by atombender** [kaiwood/vscode-endwise#1]
![endwise](https://cloud.githubusercontent.com/assets/50314/26473078/5e66580a-4177-11e7-8aa3-9b2ff863cdcb.gif)

Here's my code:

```ruby
          status = environment.status.find { |s|
            s.instance.name == name && (channel_name.nil? || s.instance.channel_name == channel_name) }
```

Hitting Enter anywhere inside the `{ ... }` block causes it to insert a line break _after_ the block. Only happens with this extension installed.

**Comment by kaiwood** [comment]
Ok, c11f4e1a3f9ae38198f2bdb0c9c991414debae6c in the latest version fixes this behavior by using proper line breaks.

Depending on the cursor position, the indentation is sometimes off +-1 space, so I leave the issue open until I figure that part out. Turns out there are a couple of edge cases to do this extension right, but we'll get there ;)

Thanks for the quick feedback, btw.!

**Comment by kaiwood** [comment]
Solved via 2be5566

**Comment by atombender** [comment]
Awesome, thanks!
